src/styles/timeline.scss

```diff
@@ -0,0 +1,58 @@
+#timeline {
```

Comment:
Rename filename into `_timeline.scss`. Import in `styles.scss` can remain the same though.

Reply:
Use class selector rather than id selector: `.timeline`
Remember to change the HTML, too.